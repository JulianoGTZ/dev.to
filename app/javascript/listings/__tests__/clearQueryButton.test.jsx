import { h } from 'preact';
import { render } from 'preact-render-spy';
import ClearQueryButton from '../elements/clearQueryButton';

describe('<ClearQueryButton />', () => {
  const defaultProps = {
    onClick: () => {
      return 'onClick';
    },
  };

  it('Should match the snapshot', () => {
    const tree = render(<ClearQueryButton {...defaultProps} />);
    expect(tree).toMatchSnapshot();
  });
});
