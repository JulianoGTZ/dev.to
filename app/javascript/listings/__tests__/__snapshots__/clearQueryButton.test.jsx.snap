// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ClearQueryButton /> Should match the snapshot 1`] = `
preact-render-spy (1 nodes)
-------
<button
  type="button"
  class="classified-search-clear"
  onClick={[Function onClick]}
>
  Ã—
</button>

`;
